README.md

```diff
@@ -13,6 +13,7 @@
   - [url](#url)
   - [type](#type)
   - [paths](#paths)
+  - [configPath](#configPath)
```

Comment:
Is it a path or a file?

Reply:
Path to the file 😄
The CLI has `--config`, but here config is already used to provide the UI5 bootstrap config...
So I decided to use the API option name, which is `configPath`. See https://sap.github.io/ui5-tooling/api/module-@ui5_project.normalizer.html

Reply:
OK if already used in the UI5 Tooling already, I am fine with it